complex/ft_main.c

```diff
@@ -41,6 +41,7 @@
 #include "fabtest.h"
 
 static int persistent = 1;
+static int do_fork = 0;
```

Comment:
I made the fork as opt-in since psm2 is the only provider that requires it (afaik) but I agree with you with regards to backward compatibility. If it isn't a lot of work to change existing scripts/config, we can have fork as opt-in. I'd prefer this since none of the other providers need fork. What do you think?

Reply:
I agree with making this opt-in, since that's the uncommon case.  We're not aiming for backwards compatibility with fabtests.  The ability to use fabtests to debug is more important IMO.

Reply:
ok, that makes sense, I am fine with this being opt-in then if we are not aiming for backwards compatibility in the fabtests.

Reply:
Thanks. I'll merge this then.